src/main/resources/assets/tropicraft/lang/en_US.lang

```diff
@@ -161,6 +161,26 @@ item.tropicraft:flower_pot.name=Flower Pot
 item.tropicraft:bamboo_door.name=Bamboo Door
 item.tropicraft:bamboo_item_frame.name=Bamboo Item Frame
 item.tropicraft:water_wand.name=Water Wand
+item.tropicraft:egg_iguana.name=Iguana Egg
```

Comment:
Shouldn't these start with "Spawn" for consistency?

Reply:
I left it as egg_thing because of compatibility with previous versions. I suppose that doesn't really matter a ton for this rewrite but...

Reply:
Sorry, I meant the translation. All vanilla eggs translate to "Spawn X", like "Spawn Zombie" etc.